lib/app/Auth/PhoenixTransactionBridge.php

```diff
@@ -22,6 +22,8 @@ public function getHeader($name) {
    * @return void
    */
   public function log($message, array $details) {
-    watchdog('PhoenixTransactionBridge', $message, $details, WATCHDOG_INFO);
+    $details = json_encode($details);
+
+    watchdog('PhoenixTransactionBridge', $message . ' !details', array('!details' => $details), WATCHDOG_INFO);
```

Comment:
use `[]` instead of `array`
it looks like that's what styleci is complaining abuot